app/js/components/tags/modal/Tag.react.spec.js

```diff
@@ -0,0 +1,35 @@
+import expect from 'expect';
+import React from 'react'; // eslint-disable-line no-unused-vars
+import {mount, shallow} from 'enzyme'; // eslint-disable-line no-unused-vars
+import Tag from './Tag.react';
+
+/**
+* @return {Func} A shallow dom for tests
+*/
+function setup(status) {
```

Comment:
Thought about it. . . I was running once, so it wasn't necessary at the time. But by all means, it could be there.